code/game/objects/items/devices/radio/radio.dm

```diff
@@ -547,10 +547,10 @@
 	name = "cyborg radio"
 	subspace_switchable = 1
 	dog_fashion = null
+	flags_2 = NO_EMP_WIRES_2
 
 /obj/item/device/radio/borg/Initialize(mapload)
 	..()

```

Comment:
I found a few of these Initialize()s that did not return the return value of ..(), I did not touch them just in case.